_posts/2023-01-30-CharlieBroderick-the-process-meme.md

```diff
@@ -0,0 +1,7 @@
+---
+title:  chapter 4 meme
+author: charlie
+image: chapter_4_meme.jpg
+chapter: 4
+---
+The base of the meme is that the operating system has to tell the program to do something
```

Comment:
Your meme is correct (good job!), but this description is incorrect. The OS allows the process to run by creating it and scheduling it for CPU time, but it doesn't tell it to do anything. A program with no instructions is not actually going to do anything except exit.